report_py3o/models/py3o_report.py

```diff
@@ -0,0 +1,360 @@
+        """Run a command to convert to our target format"""
+        filetype = self.ir_actions_report_id.py3o_filetype
+        if not Formats().get_format(filetype).native:
+            command = self._convert_single_report_cmd(
+                result_path, model_instance, data,
+            )
+            logger.debug('Running command %s', command)
+            output = subprocess.check_output(
+                command, cwd=os.path.dirname(result_path),
+            )
+            logger.debug('Output was %s', output)
+            self._cleanup_tempfiles([result_path])
+            result_path, result_filename = os.path.split(result_path)
+            result_path = os.path.join(
+                result_path, '%s.%s' % (
+                    os.path.splitext(result_filename)[0], filetype
+                )
+            )
+        return result_path
+
+    @api.multi
+    def _convert_single_report_cmd(self, result_path, model_instance, data):
+        """Return a command list suitable for use in subprocess.call"""
+        return [
+            self.env['ir.config_parameter'].get_param(
+                'py3o.conversion_command', 'libreoffice',
+            ),
+            '--headless',
+            '--convert-to',
+            self.ir_actions_report_id.py3o_filetype,
+            result_path,
+        ]
+
+    @api.multi
+    def _get_or_create_single_report(self, model_instance, data,
+                                     existing_reports_attachment):
+        self.ensure_one()
+        attachment = existing_reports_attachment.get(
+            model_instance.id)
+        if attachment and self.ir_actions_report_id.attachment_use:
+            content = base64.decodestring(attachment.datas)
+            report_file = tempfile.mktemp(
+                "." + self.ir_actions_report_id.py3o_filetype)
+            with open(report_file, "wb") as f:
+                f.write(content)
+            return report_file
+        return self._create_single_report(
+            model_instance, data)
+
+    @api.multi
+    def _zip_results(self, reports_path):
+        self.ensure_one()
+        zfname_prefix = self.ir_actions_report_id.name
+        result_path = tempfile.mktemp(suffix="zip", prefix='py3o-zip-result')
+        with ZipFile(result_path, 'w', ZIP_DEFLATED) as zf:
+            cpt = 0
+            for report in reports_path:
+                fname = "%s_%d.%s" % (
+                    zfname_prefix, cpt, report.split('.')[-1])
+                zf.write(report, fname)
+
+                cpt += 1
+        return result_path
+
+    @api.multi
+    def _merge_results(self, reports_path):
+        self.ensure_one()
+        filetype = self.ir_actions_report_id.py3o_filetype
+        if not reports_path:
+            return False, False
+        if len(reports_path) == 1:
+            return reports_path[0], filetype
+        if filetype == formats.FORMAT_PDF:
+            return self._merge_pdf(reports_path), formats.FORMAT_PDF
```

Comment:
Great job! Thank you very much.

This function does not exists anymore, but was removed in 11.0.
Function in 10.0:
https://github.com/odoo/odoo/blob/e1767490e939233c050f796f4ae4f6db627d53c3/addons/report/models/report.py#L559
Similar functionality wor QWeb starting from 11.0:
https://github.com/odoo/odoo/blob/a7cbac305ed8645ce8ce67210d2ca5d723a891f0/odoo/addons/base/ir/ir_actions_report.py#L481

As long as this is not implemented the printing for multiple records in a listview will not work.